docs/guide/adapters/README.md

```diff
@@ -133,6 +133,17 @@ _(in order of first appearance)_
     ![](../../images/cc26x2r1_dongle.jpg)
     </details>
 
+  * <details>
+    <summary>JetHome JetStick Z2/ZB6UX CC2652P1</summary>
+
+    Powerful CC2652P1 factory-made Zigbee USB dongle with external antenna.
+    * [Documentation](https://docs.jethome.ru/en/zigbee/sticks/jetstick_z2.html)
+    * [Flash instructions](https://docs.jethome.ru/en/zigbee/howto/cc_2538_bsl.html#zigbee-howto-2538-bsl)
+    * [Firmware](https://docs.jethome.ru/en/zigbee/software/ti_cc2652p.html#zigbee-software-ti-cc2652p)
```

Comment:
For the firmwares, could you directly link to my fw repo instead?

Reply:
> For the firmwares, could you directly link to my fw repo instead?

usb-dongles come pre-installed with fw from JetHome, which is freely available for download. Replacing it will void the warranty. However, there are links to your firmware on https://docs.jethome.ru/en/zigbee/software/ti_cc2652p.html#other-firmware in section "Other firmware":
```
[Zigbee2MQTT Coordinator Firmware CC2652P](https://github.com/Koenkk/Z-Stack-firmware/tree/master/coordinator/Z-Stack_3.x.0/bin/) - Firmware coordinator from the project [Zigbee2MQTT](http://zigbee2mqtt.io/).

[Zigbee2MQTT Router Firmware CC2652P](https://github.com/Koenkk/Z-Stack-firmware/tree/master/router/Z-Stack_3.x.0/bin/) - project router firmware [Zigbee2MQTT](http://zigbee2mqtt.io/).
```